drools-osgi/drools-karaf-itests/src/test/java/org/drools/karaf/itest/blueprint/KieBlueprintjBPMPersistenceKarafIntegrationTest.java

```diff
@@ -0,0 +1,169 @@
+
+    private static final String BLUEPRINT_XML_LOCATION = "/org/drools/karaf/itest/blueprint/processpersistence/kie-beans-blueprint-process-persistence.xml";
+    private static final String PERSISTENCE_XML_LOCATION = "/org/drools/karaf/itest/blueprint/processpersistence/persistence.xml";
+    private static final String DATASOURCE_XML_LOCATION = "/org/drools/karaf/itest/blueprint/processpersistence/datasource.xml";
+    private static final String KMODULE_XML_LOCATION = "/org/drools/karaf/itest/blueprint/processpersistence/kmodule.xml";
+    private static final String POM_PROPS_LOCATION = "/META-INF/maven/org.drools/drools-osgi/pom.properties";
+
+    private static final String DRL_LOCATION = "/blueprint_process_persistence/sampleRule.drl";
+    private static final String BPMN_LOCATION = "/blueprint_process_persistence/sampleProcess.bpmn2";
+
+    private static final String PROCESS_ID = "orderApproval";
+
+    @Inject
+    @Filter("(osgi.jndi.service.name=kiesession/sessionWithoutEnv)")
+    private KieSession processWithPersistenceNoEnv;
+
+    @Inject
+    @Filter("(osgi.jndi.service.name=kiesession/sessionWithEnv)")
+    private KieSession processWithPersistenceEnv;
+
+    @Test
+    public void testStartProcessNoEnv() throws Exception {
+        assertNotNull(processWithPersistenceNoEnv);
+        final ProcessInstance processInstance = processWithPersistenceNoEnv.startProcess(PROCESS_ID);
+        assertEquals("Unexpected process instance state.", ProcessInstance.STATE_ACTIVE, processInstance.getState());
+    }
+
+    @Test
+    public void testStartProcessEnv() throws Exception {
+        assertNotNull(processWithPersistenceEnv);
+        final ProcessInstance processInstance = processWithPersistenceEnv.startProcess(PROCESS_ID);
+        assertEquals("Unexpected process instance state.", ProcessInstance.STATE_ACTIVE, processInstance.getState());
+    }
+
+    @Configuration
+    public static Option[] configure() {
+        return new Option[]{
+                // Install Karaf Container
+                getKarafDistributionOption(),
+
+                // It is really nice if the container sticks around after the test so you can check the contents
+                // of the data directory when things go wrong.
+                keepRuntimeFolder(),
+                // Don't bother with local console output as it just ends up cluttering the logs
+                configureConsole().ignoreLocalConsole(),
+                // Force the log level to INFO so we have more details during the test.  It defaults to WARN.
+                logLevel(LogLevelOption.LogLevel.INFO),
+
+                // Option to be used to do remote debugging
+//                  debugConfiguration("5005", true),
+
+                // Load KIE features
+                loadKieFeatures("jndi", "transaction", "droolsjbpm-hibernate", "h2", "jbpm", "kie-aries-blueprint"),
```

Comment:
Do we have these features: `jndi` and `transaction`? I could not find those. Am I overlooking something?